setup/setup.js

```diff
@@ -0,0 +1,101 @@
+/* eslint-disable */
+"use strict";
+
+class Setup {
```

Comment:
A class is not needed here. Move stuff from constructor into module scope.

Reply:
Alright, @montogeek already mentioned this, but still wondering, what's wrong with a class?

Reply:
Nothing, it is a common practice to put your `require` at the top of the file

Reply:
A that's why, ok. But I still have some `require` statements inside my code, as these requires may or may not be called, depending on the execution flow.

Btw. I have messed something up with Git and have therefore created a new PR [here](https://github.com/webpack/webpack/pull/6880)